concrete-csprng/src/counter/mod.rs

```diff
@@ -295,6 +295,10 @@ impl<G: AesBatchedGenerator> AesCtrGenerator<G> {
         output
     }
 
+    pub fn generate_u128(&mut self) -> u128 {
+        u128::from_ne_bytes([self.generate_next(); 16])
```

Comment:
by doing `[self.generate_next(); 16]` you are creating an array of 16 `u8` with the same value for each element of the array, is this wanted ?

Reply:
Definitely not, I will fix that.